executor/adapter.go

```diff
@@ -424,9 +424,6 @@ func IsPointGetWithPKOrUniqueKeyByAutoCommit(ctx sessionctx.Context, p plan.Plan
 	case *plan.PhysicalIndexReader:
 		indexScan := v.IndexPlans[0].(*plan.PhysicalIndexScan)
 		return indexScan.IsPointGetByUniqueKey(ctx.GetSessionVars().StmtCtx), nil
-	case *plan.PhysicalIndexLookUpReader:
```

Comment:
index lookup is not allowed to use point get?

Reply:
yes

Reply:
It's more like a workaround other than a bugfix ..

Reply:
This point get is not point get executor, it's using MaxTS.

Reply:
It's a bugfix... maybe the function name is a bit confusing...
index lookup is not allowed to use max ts and this function checks whether to use max ts
https://github.com/tiancaiamao/tidb/blob/ab1abb1938782f4bd69d15650489f9074e26ade6/executor/adapter.go#L312
@zz-jason

Reply:
got it, thanks